packages/connectors/src/metaMask.ts

```diff
@@ -60,35 +56,16 @@ export function metaMask(parameters: MetaMaskParameters = {}) {
         provider.on('connect', this.onConnect.bind(this) as Listener)
     },
     async connect({ chainId, isReconnecting } = {}) {
+      if (isReconnecting && isMobileBrowser) {
+        // Prevent reconnection on mobile, it needs to be done manually via eth_requestAccount + deeplink
```

Comment:
Interesting. Can we get this to work?

Reply:
We cannot at the moment since the wallet needs to be active and online. You need to make a deeplink to wake up the wallet. We are working on updating our protocol to prevent that. The reason for that limitation is that unlike wallet connect and coinbase we don't store encryption keys locally which in turn prevents us from storing rpc message on a relayer without having the wallet online.

Reply:
I see.